control/mttrie_medium_test.go

```diff
@@ -0,0 +1,530 @@
+// +build medium
```

Comment:
Yes :) The first of 121 lines have not been changed, but they fulfill medium tests requirements.
The rest of new test cases I preferred added as a medium tests - so, there was not sense to keep the file metrics_test.go (legacy) longer.